packages/react/src/tools/a11y.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Missing IBM Copyright block in file

Side-note: would it help to name the file according to usage? Maybe: `src/tools/keyboard-navigation.js`